lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -48,7 +48,7 @@ def search(query, language: nil, replacements: {}, subauth: nil, context: false,
         def load_graph(url:)
           access_start_dt = Time.now.utc
 
-          @graph = graph_service.load_graph(url: url)
+          @full_graph = graph_service.load_graph(url: url)
```

Comment:
I separated out full_graph from filtered_graph to be consistent with how these are treated in find_term.